Makefile

```diff
@@ -21,8 +21,10 @@ API_GO_HEADER_FILE:=pkg/apis/header.go.txt
 
 # Container image-related variables
 DOCKERFILE=Dockerfile
-IMAGE_TAG=openshift/origin-cluster-node-tuning-operator
-IMAGE_REGISTRY=quay.io
+REGISTRY=quay.io
+ORG=openshift
+TAG=$(shell git branch | grep \* | cut -d ' ' -f2)
```

Comment:
Do we need to use so many filters?  Wouldn't `git rev-parse --abbrev-ref HEAD` work?

Reply:
for consistency mostly, this is how is being done in other repos

Reply:
To replace one command with three is completely unnecessary and error prone.